poolrpc/trader.proto

```diff
@@ -1011,6 +1018,35 @@ message RecoverAccountsResponse {
     uint32 num_recovered_accounts = 1;
 }
 
+message AccountModificationFeesRequest {
+}
+
+message AccountModificationFee {
+    // Modification action type.
+    string action = 1;
+
+    // Action transaction block height.
+    int32 block_height = 2;
+
+    // Action transaction timestamp.
+    int64 timestamp = 3;
+
+    // Action transaction output amount.
+    int64 output_amount = 4;
+
+    // Action transaction fee.
+    int64 fee = 5;
+}
+
+message ListOfAccountModificationFees {
+    repeated AccountModificationFee account_modification_fee = 1;
+}
+
+message AccountModificationFeesResponse {
+    // A map from account key to an ordered list of account modification fees.
+    map<string, ListOfAccountModificationFees> account_modification_fees = 1;
```

Comment:
nit: call this just `accounts` and the above message `modification_fees`?
Then on the CLI the JSON looks like this:
```
{
  "accounts": {
    "03xxxx": {
      "modification_fees": [
        { ... }
      ]
    }
  }
}
```